### ISSUE TITLE
Error with OnDraw callback for new OpenCV  module

### ISSUE THREAD

**Issue by MargaretAN9** [publiclab/image-sequencer#1155]
### Please describe the problem (or idea)

We are trying to use openCV to do image processing, however it seems we get an error on the OnDraw callback looking for a way to update the UI.  Does each module need to have its own step process for the UI or can that get be avoided all together.   

`image-sequencer.js:100409 Uncaught TypeError: Cannot read property 'onDraw' of undefined
`
> What did you expect to see that you didn't?

### Please show us where to look
```

     for (var util in getStepUtils) {
        if (getStepUtils.hasOwnProperty(util)) {
          step[util] = getStepUtils[util];
        }
      }

      // Tell UI that a step is being drawn.
      step.UI.onDraw(step.options.step);

      // provides a set of standard tools for each step
      var inputForNextStep = require('./RunToolkit')(ref.copy(input));

```

### What's your PublicLab.org username?

> maggpi

### Browser, version, and operating system

> Windows 10, Chromium Version 73

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by jywarren** [comment]
Hi! Thanks for opening this here! Is this in reference to @aashna27's blob detection module in #1120 ?

**Comment by MargaretAN9** [comment]
Yes, I am trying to convert the IS data uri to an OpenCV matrix  (and vice versa) format using similar methods as @aashna27.   It looks like I am having different issues (on Draw error) than  #1120  but I am also still learning how IS works.   In particular , the purpose and operation of 'step' utililities?
